docs/sdk/src/guides/your_first_node.rs

```diff
@@ -1 +1,117 @@
 //! # Your first Node
+//!
+//! In this guide, you will learn how to run a runtime, possibly generated in
+//! [`your_first_runtime`], in a node. Within the context of this guide, we will focus on running a
+//! solo-chain node.
+//!
+//! First, make sure to read all the content covered in
+//! [`crate::reference_docs::node_and_chain_spec`]. Once done, you should be familiar with the two
+//! main options of launching a node:
+//!
+//! 1. A template
+//! 2. An Omni Node
+//!
+//! While we will not do it here, it should be a great learning step to try and integrate the
+//! runtime built in this guide thus far into the [`minimal_template_runtime`], and then run it with
+//! `minimal_template_node`. This will be the former approach.o.
```

Comment:
Can you maybe explain what the omni node is and why it is nicer than a template node for quick start?`